card/src/action/card.action.js

```diff
@@ -0,0 +1,35 @@
+export function cardRequesting() {
+  return {
+    type: "CARD_REQUESTING",
```

Comment:
store in variable and make use of it
ex:- export const CARD_REQUESTING="CARD_REQUESTING"

 type: CARD_REQUESTING

Reply:
Done